SwanPortAllocator/swanportallocator/__init__.py

```diff
@@ -4,7 +4,7 @@ def _jupyter_nbextension_paths():
     # Empty to avoid error when automatically trying to enable all nbextensions
     return []
 
-def _jupyter_server_extension_paths():
+def _jupyter_server_extension_points():
```

Comment:
Does this change match the commit message?

Reply:
Yes.